__tests__/UglifyJsPlugin.test.js

```diff
@@ -0,0 +1,824 @@
+"use strict";
```

Comment:
The `__snapshots__` directory is just in there as a 'where are we' orientation point, `jest` will create a separate `snapshot` for each `${name}.test.js` in `__snapshots__`. The important part is to split test into logical groups, so particular parts can be tested more independently